src/components/PatentUpload/PatentUpload.js

```diff
@@ -1,4 +1,96 @@
-const PatentUpload = () => (
-    <div>Patent Upload Page</div>
-);
+import StringManager from '../../utils/StringManager';
+import Storage from "@aws-amplify/storage";
+import { Component } from 'react';
+import { Form, Input } from 'antd';
+import './PatentUpload.css';
+
+class PatentUpload extends Component {
+
+    // Set initial state to null
+    state = {
+        proteinName: "",
+        fileName: "",
+        pdfFile: "",
+        response: ""
+      };
+    
+    // Call Storage function to store in S3 bucket
+    // respond if success or failure
+    uploadFile = () => {
+    // TODO: S3 Bucket config SetS3Config("", "protected");
+    Storage.put(`${this.upload.value[0]}/${this.upload.files[0].name}`,
+            this.upload.files[0],
+            { contentType: this.upload.files[0].type })
+        .then(result => {
+        this.upload = null;
+        this.setState({ response: "Success uploading file!" });
+        })
+        .catch(err => {
+        this.setState({ response: `Cannot upload file: ${err}` });
+        });
+      };
+    
+
+    // Save file name and file in an array
+    render() {
+        return (
+            <div className="App">
+            <h2>Upload Patents</h2>
+            <Form
+                name="basic"
+                className="protein-container"
+                initialValues={{ remember: true }}
+                onChange={e =>
+                    this.setState({
+                        proteinName: this.upload.value[0]
+                    })
+                }
+            >
+                <Form.Item
+                    className="protein-container_input"
+                    label={StringManager.get('typeProtein')}
+                    name="protein"
+                    rules={[{ required: false }]}
+                >
+                    <Input />
+                </Form.Item>
+            </Form>
+            <input
```

Comment:
Ideally we should not need to create any html element and leverage ant design. See some options here:
https://ant.design/components/input/